model/insights/insights.yaml

```diff
@@ -0,0 +1,26 @@
+components:
+  schemas:
+    Insights:
+      type: object
+      description: Configuration of insights providing methods
+      properties:
+        port_mirroring:
+          $ref: "#/components/schemas/Insights.PortMirroring"
+
+    Insights.PortMirroring:
+      type: array
+      items: 
+        $ref: "#/components/schemas/Insights.PortMirroring.Config"
+
+    Insights.PortMirroring.Config:
+      type: object
+      properties:
+        name:
+          type: string
+          description: Uniquely identifies a config entry
+        ingress_port_name:
```

Comment:
Similarly here, these port names are implementation dependent (as they're auto-generated by the server implementation)